### PR TITLE
MEPTS-1255 | MI CAT 15 Changes For June Release

### PR DESCRIPTION
**Pull request by Emmanuel0077**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/IntensiveMonitoringCohortQueries.java [hunk 1]
```diff
@@ -3542,4 +3542,143 @@ public CohortDefinition getPatientsOnMICat18Numerator() {
...
+
+    String query =
+            " SELECT p.patient_id "
+                    + "FROM   patient p "
+                    + "       inner join encounter e "
+                    + "               ON e.patient_id = p.patient_id "
+                    + "       inner join obs o "
+                    + "               ON o.encounter_id = e.encounter_id "
+                    + "WHERE  e.encounter_type = ${6} "
+                    + "       AND ( o.concept_id = 2015 "
```
Comment (iteliolucas) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/IntensiveMonitoringCohortQueries.java]: 2015 is not mapped, please add the corresponding mapping

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/IntensiveMonitoringCohortQueries.java [hunk 2]
```diff
@@ -3542,4 +3542,143 @@ public CohortDefinition getPatientsOnMICat18Numerator() {
...
+                    + "             AND o.value_coded IN ( 23748, 6293, 23749, 29, "
```
Comment (iteliolucas) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/IntensiveMonitoringCohortQueries.java]: 23748, 6293, 23749, 29 and so on are not mapped, please add the corresponding mapping

**Comment by Emmanuel0077** [comment]
Please, consider the last commit, these changes were removed because these
queries already exited in the code

Itélio Lucas ***@***.***> escreveu em sex., 26/05/2023 às
10:54 :

**Comment by Emmanuel0077** [comment]
The correct mapping was made in the second commit


Emmanuel Vicente ***@***.***> escreveu em sex., 26/05/2023
às 10:57 :

**Comment by iteliolucas** [comment]
Looks ok

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java [hunk 3]
```diff
@@ -9295,7 +9319,7 @@ public CohortDefinition getMI15Den15() {
         EptsReportUtils.map(
```
Comment (eliazmuchave) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java]: This definition is no longer used, remove it from cohort search

**Review by eliazmuchave — CHANGES_REQUESTED** [review]

**Review by eliazmuchave — APPROVED** [review]
Looks good to me

**Pull request merged**